layers/meta-resin-raspberrypi/recipes-bsp/u-blox-modeswitch/files/u-blox-switch.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+while true; do
+        echo -e "AT+UUSBCONF=2,\"ECM\"\r\n" > /dev/ttyACM0
+        sleep 2
```

Comment:
How did we decide on these timeouts?

Reply:
The 2s timeout is probably not necessary, but the 20s timeout is necessary for the modem to restart and show up again on the USB bus.